### ISSUE TITLE
Not using *.dat-files as cache

### ISSUE THREAD

**Issue by LukasCBossert** [monperrus/bibtexbrowser#128]
My setup is that I have the bibtexbrowser-files on a server and I reuse this "instance" for displaying a bibliography on different pages. 
For this I load a *.bib-file from a GitLab-Page, where this is daily updated.

However the updated bibliography is not shown in the bibtexbrowser, since on the server there is a (or multiple) file like 
`bibtexbrowser_013bd0f50633d58f93e199ecd0cb442c.dat` generated. It seems like this works as a backup/cache. 

How can I get rid of those .dat-files in the sense that bibtexbrowser would always load the (update) remote .bib-file?

UPDATE: 

I see now the part responsible for creating the dat-files. 
```

  // ---------------------------- HANDLING caching of compiled bibtex files
  // for sake of performance, once the bibtex file is parsed
  // we try to save a "compiled" in a txt file
  $compiledbib = 'bibtexbrowser_'.md5($bibtex_filenames).'.dat';

```

What can I add in the `bibtexbrowser.local.php` to prevent this?

**Comment by monperrus** [comment]
do you mean you load the bib via an HTTP url?

**Comment by LukasCBossert** [comment]
Yes, I found out that commenting out `$compiledbib = 'bibtexbrowser_'.md5($bibtex_filenames).'.dat';` bibtexbrowser will always load the bibfile from the url-source. But I am not sure if this has any side effect.s

**Comment by monperrus** [comment]
OK. Just added support for this:
https://github.com/monperrus/bibtexbrowser/pull/129

Does it work for you?

**Comment by LukasCBossert** [comment]
Thanks for the quick feature request-add in. As far as I can see this works fine.

**Comment by LukasCBossert** [comment]
@monperrus I was wrong. when I use the code I now get an empty page - nothing is shown.

**Comment by monperrus** [comment]
strange, it works for me. what's the error in /var/log/apache2/error.log?

**Comment by LukasCBossert** [comment]
Yes, it works - I had something wrong on my end (and deleted the comment then).

**Comment by monperrus** [comment]
ok, thanks!

**Comment by monperrus** [comment]
closed by #129
